src/main/java/mate/academy/security/AuthenticationServiceImpl.java

```diff
@@ -2,19 +2,18 @@
 
 import java.util.Optional;
 import mate.academy.exception.AuthenticationException;
-import mate.academy.lib.Inject;
-import mate.academy.lib.Service;
+import mate.academy.lib.Injector;
 import mate.academy.model.User;
 import mate.academy.service.ShoppingCartService;
 import mate.academy.service.UserService;
 import mate.academy.util.HashUtil;
 
-@Service
 public class AuthenticationServiceImpl implements AuthenticationService {
-    @Inject
-    private UserService userService;
-    @Inject
-    private ShoppingCartService shoppingCartService;
+    private static final Injector injector = Injector.getInstance("mate.academy");
```

Comment:
why did you change Inject here?

Reply:
I changed Inject, because its didn't work with annotations, getting error like:
![image](https://user-images.githubusercontent.com/61744468/142597799-c4cc2fd9-30f0-4939-855a-fe32e1710e06.png)